decidim-generators/lib/decidim/generators.rb

```diff
@@ -6,7 +6,7 @@ module Decidim
   module Generators
     def self.edge_git_branch
       if Decidim::Generators.version.match?(/\.dev$/)
-        "develop"
+        "chore/rails-upgrade"
```

Comment:
```suggestion
        "develop"
```

Reply:
https://github.com/orgs/decidim/teams/maintainers this needs to be changed after your review.